### PR TITLE
Allow removal of wildcard ip whitelist

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Allow removal of whildcard ip whitelist

Related issues:
- https://github.com/ainblockchain/ain-blockchain/issues/811

### CODE REVIEW

File: json_rpc/api-access.js [hunk 1]
```diff
@@ -90,6 +90,19 @@ module.exports = function getApiAccessApis(node) {
         done({ code: 403, message: 'Forbidden' });
         return;
       }
+      if (CommonUtil.isWildcard(args.message.ip)
+          && NodeConfigs.DEV_CLIENT_API_IP_WHITELIST === '*') {
```
Comment (liayoo) [inline json_rpc/api-access.js:94]: isWildcard can be used here as well?

**Review by minsulee2 — APPROVED** [review]
LGTM tho, I've got a question that we only support ip address removal?

**Review by liayoo — APPROVED** [review]
LGTM with a comment. Thanks!

**Comment by platfowner** [comment]
> LGTM tho, I've got a question that we only support ip address removal?

we support ip addresses plus wildcard. any others in you mind?

File: json_rpc/api-access.js (hunk 1 above)
Comment (platfowner) [reply json_rpc/api-access.js]: done, thanks!

**Comment by minsulee2** [comment]
> > LGTM tho, I've got a question that we only support ip address removal?
> 
> we support ip addresses plus wildcard. any others in you mind?

Nope! I was just wondering thanks!!!

**Comment by platfowner** [comment]
Thanks for the fast review, now merging.. @minsulee2 @liayoo

**Pull request merged**